### ISSUE TITLE
Links in code invisible

### ISSUE THREAD

**Issue by howtonotwin** [MinecraftForge/Documentation#71]
Given the Markdown snippet `[`test`][test]`, mkdocs produces the html `<a><code>test</code></a>`. As `code` overrides the blue color of `a` with its own orange, the resulting link (![Above snippet rendered](https://cloud.githubusercontent.com/assets/15616055/19669312/5ace3f98-9a2c-11e6-8e92-4d0e1e1a2ad9.PNG)) is practically invisible.

Adding the following to `style_fixes.css`:

``` css
/* Color code inside links. This makes links like [`test`][test] visible. */
a code {
    color: inherit;
}
```

results in ![Above snippet rendered with css fix](https://cloud.githubusercontent.com/assets/15616055/19669400/0a955812-9a2d-11e6-9494-df33a632dc29.PNG).

**Comment by tterrag1098** [comment]
The entire site is being restyled soon (cc @PaleoCrafter) so this is probably irrelevant. Might be worth a quick fix though. Up to him.

**Comment by tterrag1098** [comment]
Closed via #113
